pkgs/top-level/all-packages.nix

```diff
@@ -4185,7 +4185,7 @@ in
   };
 
   diffoscope = diffoscopeMinimal.override {
-    enableBloat = true;
+    enableBloat = !stdenv.isDarwin;
```

Comment:
Wouldn't it then be equal to diffoscopeMinimal?

Reply:
yes, I think that would preferable instead of keeping a broken package on darwin